worker/worker.py

```diff
@@ -104,13 +107,24 @@ def _get_memory_bytes(self):
             # Fallback to sysctl when os.sysconf('SC_PHYS_PAGES') fails on OS X
             return int(check_output(['sysctl', '-n', 'hw.memsize']).strip())
 
+    def _get_gpu_count(self):
+        if not self._docker._use_nvidia_docker:
+            return 0
+
+        container_id = self._docker.run_nvidia_smi('-L', 'gcr.io/tensorflow/tensorflow:latest-gpu')
```

Comment:
I think we can use a more basic image than the tensorflow one? Maybe one of these:
https://hub.docker.com/r/nvidia/cuda/